setup.py

```diff
@@ -20,24 +20,27 @@ def readme():
     install_requires=[
         "numpy>=1.15.0",
         "packaging>=19",
-        "larq>=0.9.2,<0.14.0",
-        "zookeeper>=1.0.0,<1.4.0",
+        "larq>=0.9.2,<0.13.2",
```

Comment:
Let's try with the latest Larq instead:
```suggestion
        "larq>=0.9.2",
```